### PR TITLE
Added interctive-shell option to pulp-cli.

### LINKED ISSUE #181: Feature: add a "pulp shell" command to start an interactive command-interpreter

### PR DESCRIPTION
**Pull request by ggainey**
closes #181

### CODE REVIEW

File: tests/settings.toml.example [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [cli]
-base_url = "http://localhost:8080"
+base_url = "http://localhost"
```
Comment (gerrod3) [inline tests/settings.toml.example:2]: Accidental change? The CI uses port 8080 I'm pretty sure.

Comment (mdellweg) [reply tests/settings.toml.example:2]: Right. And this file is simply copied to satisfy the CI.

Comment (ggainey) [reply tests/settings.toml.example]: EDEV_IS_IDIOT raised.  Fixing.

**Comment by ggainey** [comment]
> After experimenting with this, i would prefer if starting the shell were a little bit more explicit. Like `pulp shell`.

That will take some experimentation, I think. Will try to pry up some time.

**Comment by ggainey** [comment]
Closing this in preference for #187

**Pull request closed without merging**